tests/garage/tf/policies/test_categorical_mlp_policy_with_model.py

```diff
@@ -0,0 +1,77 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.models import Model
+from garage.tf.policies import CategoricalMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscreteEnv
+
+
+class SimpleMLPModel(Model):
+    """Simple MLPModel for testing."""
+
+    def __init__(self, name, output_dim, *args, **kwargs):
```

Comment:
why *args and *kwargs?

Reply:
because the policy will pass other arguments like `hidden_sizes` and `hidden_nonlinearity` to the model, but in this `SimpleMLPModel`, I don't want to deal with these.